quipucords/quipucords/settings.py

```diff
@@ -436,8 +435,8 @@ def warn_on_deprecation(message, category, filename, lineno, file=None, line=Non
     logger.warning(f"{message} (from {filename}:{lineno})")
 
 
-warnings.showwarning = warn_on_deprecation
-warnings.simplefilter("always", DeprecationWarning)
+# warnings.showwarning = warn_on_deprecation
+# warnings.simplefilter("always", DeprecationWarning)
```

Comment:
Don't forget to un-comment these.

Reply:
I rebased on top of the changes you made that fix this issue. Thanks!